resources/email.py

```diff
@@ -0,0 +1,44 @@
+from flask import Flask
+from flask_restful import Resource, reqparse
+from flask_mail import Message
+from flask_jwt_extended import jwt_required, get_jwt_claims
+import app
+from models.user import UserModel
+
+class Email(Resource):
+    parser = reqparse.RequestParser()
+    parser.add_argument('userid')
+    parser.add_argument('title') 
+    parser.add_argument('body') 
+
+    @jwt_required
+    def post(self):
+        check if is_admin exist if not discontinue function
```

Comment:
I think this should be a comment like in user.py line 32? Getting a build error here

Reply:
Got it! Thanks